grpc-interface-support/data/templates/python/ServiceNameServiceServerFactory.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import grpc
+import concurrent.futures
+
+from ${{ service_name_lower }}_service_sdk.${{ service_name_lower }}_pb2_grpc import (
+    ${{ service_name }}Servicer, add_${{ service_name }}Servicer_to_server
```

Comment:
why are we using here `Servicer` and not `Service` or `Client` and `Server`

Reply:
Without much knowledge here but "Servicer" seems to be the common naming here in python x grpc:
https://grpc.io/docs/languages/python/basics/

Reply:
I see, yeah then its fine :)